CHANGELOG.md

```diff
@@ -110,6 +111,15 @@
 - [v0.2.0](#v020)
 - [v0.1.0](#v010)
 
+## [v1.46.1]
+> Release date: 2025/04/02
+
+### Fixed
+- Fixed an issue where entities were replaced silently when ID was present in input files and select_tags didn't match.
+[#1572](https://github.com/Kong/deck/pull/1572)
+[go-database-reconciler #](https://github.com/Kong/go-database-reconciler/pull/214)
```

Comment:
PR numbers are left out here in [] and in the line below.

Reply:
I think you took inspiration from last few releases where I forgot too. 😂 
I will add a PR to fix those.

Reply:
I've added for the last release as well

Reply:
I'll rebase after that's merged